packages/oid4vci-demo-frontend/src/ecosystem-config.ts

```diff
@@ -157,15 +157,22 @@ export interface SSIInformationRequestPageConfig extends PageConfig {
 export type DataFormRow = DataFormElement[];
 
 export interface DataFormElement {
-    id: string;
-    title: string;
-    key: string;
-    type: DataFormInputType;
-    required: boolean;
+    id: string
+    key: string
+    type: HTMLInputTypeAttribute
+    required?: boolean
     defaultValue?: string
+    label?: string
+    // FIXME can be placed better but for now this was the easiest. when there is a url for the label (and this is only possible for a checkbox) we use an element for the label to be displayed
+    labelUrl?: LabelUrl
```

Comment:
fine for now, next time we have to display another checkbox and another link it's maybe nice to support Markdown in the translation.json

Reply:
ok, i was not aware of this power. i changed it to use markup. still not happy about the implementation cause i don't think dangerouslySetInnerHTML should be used. but it does cleanup some messy code

Reply:
As long as we are in control and not an end-user about what goes in, inner HTML should not be a problem.

So external input => never allow the inner HTML option. We control it => ideally do not set it.
The only reason for the ideally is that you will have to know up front that there never will be external input. Which for translations could come from using variables for user input